src/lib/viewers/media/VideoBaseViewer.js

```diff
@@ -43,10 +43,11 @@ class VideoBaseViewer extends MediaBaseViewer {
         this.mediaEl.setAttribute('playsinline', '');
 
         // Play button
-        this.playButtonEl = this.mediaContainerEl.appendChild(document.createElement('div'));
+        this.playButtonEl = this.mediaContainerEl.appendChild(document.createElement('button'));
```

Comment:
Do we need to add any styling resets with this change? Buttons often have default styles.